src/main/java/com/moviecat/model/Genre.java

```diff
@@ -36,13 +35,17 @@
     public Genre() {
     }
 
+    /*
     @JsonCreator
     public Genre(@JsonProperty String title, @JsonProperty String description) {
         this.title = title;
         this.description = description;
     }
 
-    public Genre(String id, String title, String description) {
+     */
```

Comment:
Ooh great eyesight. Please help me remove the unused constructor. I forgot to remove it.